depthai.py

```diff
@@ -180,20 +175,25 @@ def show_mobilenet_ssd(entries_prev, frame, is_depth=0):
             if e[0]['label'] > len(labels):
                 print("Label index=",e[0]['label'], "is out of range. Not applying text to rectangle.")
             else:
-                pt_t1 = x1, y1 + 20
-                cv2.putText(frame, labels[int(e[0]['label'])], pt_t1, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color, 2)
-
-                pt_t2 = x1, y1 + 40
-                cv2.putText(frame, '{:.2f}'.format(100*e[0]['confidence']) + ' %', pt_t2, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
+                pt_t1 = x1 + 6, y1 -10
+                bg_t1 = x1, y1 -28
+                bg_t2 = x1  + 78, y1
+                cv2.rectangle(frame, bg_t1, bg_t2, color, cv2.FILLED)
+                color = (0, 0, 0) # bgr
+                cv2.putText(frame, labels[int(e[0]['label'])], pt_t1, cv2.FONT_HERSHEY_SIMPLEX, 0.6, color, 1)
+
+                pt_t2 = x1 + 90, y1 - 10
+                cv2.putText(frame, '{:.2f}%'.format(100*e[0]['confidence']), pt_t2, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
+                color = (0, 0, 255) # bgr
                 if config['ai']['calc_dist_to_bb']:
-                    pt_t3 = x1, y1 + 60
-                    cv2.putText(frame, 'x:' '{:7.3f}'.format(e[0]['distance_x']) + ' m', pt_t3, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
+                    pt_t3 = x1 + 5, y1 + 20
+                    cv2.putText(frame, 'x: {:4.1f}m'.format(e[0]['distance_x']), pt_t3, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
 
-                    pt_t4 = x1, y1 + 80
-                    cv2.putText(frame, 'y:' '{:7.3f}'.format(e[0]['distance_y']) + ' m', pt_t4, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
+                    pt_t4 = x1 + 5, y1 + 40
+                    cv2.putText(frame, 'y: {:4.1f}m'.format(e[0]['distance_y']), pt_t4, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
 
-                    pt_t5 = x1, y1 + 100
-                    cv2.putText(frame, 'z:' '{:7.3f}'.format(e[0]['distance_z']) + ' m', pt_t5, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
+                    pt_t5 = x1 + 5, y1 + 60
+                    cv2.putText(frame, 'z: {:4.1f}m'.format(e[0]['distance_z']), pt_t5, cv2.FONT_HERSHEY_SIMPLEX, 0.5, color)
```

Comment:
`FONT_HERSHEY_TRIPLEX` instead of `FONT_HERSHEY_SIMPLEX` looks better in my opinion, what do you think @philnelson ?

Reply:
Hmm I don't think any of them look great to be honest, I may test replacing them with some open source opentype or truetype font... I'll do a quick test of SIMPLEX .vs. TRIPLEX and post here.

Reply:
FONT_HERSHEY_SIMPLEX:
<img width="412" alt="Screen Shot 2020-05-06 at 5 05 46 PM" src="https://user-images.githubusercontent.com/22225/81240346-093f7a00-8fbc-11ea-966e-7f0c78408b32.png">

FONT_HERSHEY_TRIPLEX:
<img width="412" alt="Screen Shot 2020-05-06 at 5 06 34 PM" src="https://user-images.githubusercontent.com/22225/81240364-10ff1e80-8fbc-11ea-8dc9-0f5e73cb0434.png">

Reply:
I vote for SIMPLEX.  Clearer and simpler.